memory-viz/bin/cli.js

```diff
@@ -73,7 +73,8 @@ program
         "path to a CSS file containing global styles for the SVG",
         parseGlobalStyle
     )
-    .option("-t, --theme <name>", "use themed styling for the generated SVG");
+    .option("-t, --theme <name>", "use themed styling for the generated SVG")
+    .option("--not-interactive", "disable hover interactivity for object IDs");
```

Comment:
You can name this `--no-interactive` instead. This is both more conventional and will hook into commander's built-in boolean option processing (see https://github.com/tj/commander.js?tab=readme-ov-file#other-option-types-negatable-boolean-and-booleanvalue).

Also, make sure to have the CLI documentation align with this as well (i.e., the option documented should be `--no-interactive`, not `--interactive`).